types/parse5/v4/tslint.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I chose to disable failing lint rules instead of trying to fix the problem spots. This was done in favor of keeping the v4 types _exactly_ as they were when I copied and pasted them from their originating repo.